com.jgraph.drawio.desktop.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "com.jgraph.drawio.desktop",
+  "branch": "stable",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "draw.io",
+  "separate-locales": false,
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=home",
+    "--share=ipc",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "draw.io",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar -x draw.io-*.deb",
+        "rm -f draw.io-*.deb",
+        "tar -xf data.tar.xz",
+        "cp -r opt/* /app",
+        "install -Dm755 draw.io.sh /app/bin/draw.io",
+        "install -Dm644 usr/share/applications/draw.io.desktop /app/share/applications/com.jgraph.drawio.desktop",
```

Comment:
I believe that this has to be com.jgraph.drawio.desktop.desktop

Reply:
Done